addons/website_slides/tests/test_gamification_karma.py

```diff
@@ -98,8 +98,6 @@ def test_karma_gain(self):
 
         # Leave the finished course
```

Comment:
Would modify to
`# Leave the finished course - karma should not move as we only archive membership`